crates/sequencer/src/checkpoint/expiry.rs

```diff
@@ -11,6 +11,7 @@ use tracing::{debug, error, warn};
 
 use crate::checkpoint::CheckpointHandle;
 
+// FIXME WHAT DO THESE FIELDS REPRESENT????????
```

Comment:
See what missing docs do? 😂

Reply:
Now you're left to resort to gitblame Spanish Inquisition.

Reply:
fineeeee :P